contrib/build-flatpak.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+export LC_ALL=C
+set -e
+SECONDS=0
+
```

Comment:
A small documentation on how to install the requirements(SDK, Platform), where the final build artifacts are and how to install it would be nice.